### DISCUSSION TITLE
what is the difference to zagjs?

### DISCUSSION THREAD

**Discussion by roccho-dev** [ai/nanostores#240] [General]
What is the difference to zagjs?
https://zagjs.com/

Why anyone has chosen nanostores instead of zagjs?
Maybe, we have to check these. Please let me know your thought.
https://codesandbox.io/s/zustand-valtio-jotai-counters-my6np

**Comment by ai** [comment]
Zagis is state machine, while Nano Stores is using stores and computed stores.
